README.md

```diff
@@ -75,6 +81,80 @@ This assumes you have the correct ssh key in your ssh-agent. If you like to give
 $ docker compose up -d
 ```
 
+### Deploy Serve for local development with Rancher Desktop
+
+Start with instructions in [Serve Charts > How to Deploy](https://github.com/ScilifelabDataCentre/serve-charts?tab=readme-ov-file#how-to-deploy) and come back here when you get to the point of building the studio image.
+
+Again, this setup assumes you have [Rancher Desktop](https://rancherdesktop.io/) installed and running.
+
+```bash
+$ git clone git@github.com:ScilifelabDataCentre/stackn.git
+$ cd stackn
+$ git checkout develop
+$ cp .env.template .env
+$ cp ~/.kube/config cluster.conf
+$ cp ~/.ssh/id_rsa.pub id_rsa.pub
+$ cat Dockerfile local.Dockerfile > local_.dockerfile
+$ nerdctl build --namespace k8s.io -t mystudio -f local_.dockerfile .
+```
+
+Now continue setting up serve charts until you get to the PyCharm setup.
+
+##### PyCharm setup
+
+Just commands:
+
+1. Do this weirdness due to [this](https://youtrack.jetbrains.com/issue/PY-55338/Connection-to-python-console-refused-with-docker-interpreter-on-Linux)
+    1. go to Help | Find Action | Registry
+    2. disable python.use.targets.api
+    3. recreate the interpreter from scratch
+2. Setup ssh interpreter
+```bash
+# This will open ssh connection from the pod to our host machine
+# Because we made everything super NOT secure for local development you can ssh there without password and as root
+$ sudo kubectl port-forward svc/serve-studio 22:22
+```
+3. Set up the interpreter in PyCharm
+    1. Go to File | Settings | Project: stackn | Python Interpreter
```

Comment:
For me this was actually PyCharm | Settings | Project: stackn | Python Interpreter